WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/WatchDogView.java

```diff
@@ -197,9 +226,9 @@ private JFreeChart createPieChart(final PieDataset dataset,
 	}
 
 	private JFreeChart createBarChart(final DefaultCategoryDataset dataset,
-			final String title) {
```

Comment:
In the original version (before refactoring), String title was marked as final, so I've followed this logic. I guess the responsibility of this method is to create Bar Chart in a way a user of the method requested, so it shouldn't change anything.